Cargo.toml

```diff
@@ -48,3 +48,7 @@ features = [
 [features]
 # Enable all API, even ones not available on all OSs.
 all = []
+
+[dev-dependencies]
```

Comment:
Please don't add (dev) dependencies.

Reply:
you mean don't add `quickcheck` at all or add it ad a regular prod dependency? It is for testing only.

Reply:
I mean either, socket2 is used by a lot of crates, introducing a new dependency takes to properly fed it, which I don't have a the moment. Furthermore the added befit is slim.

Reply:
quickcheck tests and dev dependency removed.